modules/rosa-adding-node-labels.adoc

```diff
@@ -37,11 +50,50 @@ $ rosa create machinepool --cluster=mycluster --name=db-nodes-mp --replicas=2 --
 [source,terminal]
 ----
 I: Machine pool 'db-nodes-mp' created successfully on cluster 'mycluster'
+I: To view all machine pools, run 'rosa list machinepools -c mycluster'
+----
+
+* To add or update node labels in an existing a machine pool:
+. List the machine pools in the cluster:
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID           AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS    TAINTS    AVAILABILITY ZONES    SPOT INSTANCES
+Default      No           2         m5.xlarge                          us-east-1a            N/A
+db-nodes-mp  No           2         m5.xlarge                          us-east-1a            No
+----
+
+. Add or update the node labels for a machine pool:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> --replicas=<replica_count> --labels=<key>=<value>,<key>=<value> <machine_pool_id> <1>
```

Comment:
The above doc had mentioned how to update replicas, I don't think it's reasonable to show the replicas editing in the command again. Especially may cause customer confusion that they have to update the replicas when they only want to update lables.